src/components/HeaderWithCloseButton.js

```diff
@@ -78,6 +81,25 @@ const propTypes = {
         total: PropTypes.number,
     }),
 
+    /** Whether we should show an avatar */
+    shouldShowAvatarWithDisplay: PropTypes.bool,
+
+    /** Report, if we're showing the details for one and using AvatarWithDisplay */
+    report: iouReportPropTypes,
+
+    /** Policies, if we're showing the details for a report and need info about it for AvatarWithDisplay */
+    policies: PropTypes.shape({
+        /** Name of the policy */
+        name: PropTypes.string,
+    }),
+
+    /** Policies, if we're showing the details for a report and need participant details for AvatarWithDisplay */
+    personalDetails: PropTypes.objectOf(participantPropTypes),
+
+    /** Additional styles to render on the container of this component */
+    // eslint-disable-next-line react/forbid-prop-types
+    containerStyles: PropTypes.arrayOf(PropTypes.object),
```

Comment:
NAB but if these are only used when `shouldShowAvatarWithDisplay` is `true` then maybe that's a case for having a completely separate component `HeaderWithCloseButtonAndAvavtar` or something like that 🤷.